src/admin/content/content.const.ts

```diff
@@ -22,18 +28,37 @@ export const CONTENT_PATH = {
 	CONTENT_EDIT: `/${ROUTE_PARTS.admin}/${ROUTE_PARTS.content}/:id/${ROUTE_PARTS.edit}`,
 };
 
+export const ITEMS_PER_PAGE = 10;
+
 export const CONTENT_OVERVIEW_TABLE_COLS: TableColumn[] = [
 	{ id: 'title', label: i18n.t('admin/content/content___titel') },
 	{ id: 'content_type', label: i18n.t('admin/content/content___content-type') },
 	{ id: 'author', label: i18n.t('admin/content/content___auteur') },
```

Comment:
is het veel moeite om title, content_type en author ook sortable te maken? of was dat niet gevraagd in de specs?

Reply:
Is idd niet expliciet in het ticket gedefiniëerd wat allemaal sortable moest zijn.
Die velden zijn het (voorlopig) geworden in samenspraak met Sander. 
Eventueel iets om tijdens een demo te vragen 👍